src/core/CoreSession.jl

```diff
@@ -7,7 +7,7 @@ mutable struct  CoreSession <: AbstractCoreSession
     database::CoreDatabase
     sessionID::Bytes
     transactions::Dict{UUID,T} where {T<:Union{Nothing,<:AbstractCoreTransaction}}
-    type::Int
+    type::Int32
```

Comment:
Nice! I found it from https://developers.google.com/protocol-buffers/docs/proto#enum

> Enumerator constants must be in the range of a 32-bit integer.

Reply:
But here it should be our self definied EnumType which is a Int32! :-)